### ISSUE TITLE
Improve Lighthouse score

### ISSUE THREAD

**Issue by CrocoDillon** [CrocoDillon/browsersupport.io#26]
Current score:

- **Performance 86**
  - [ ] Improve perceptual speed index ([learn more](https://developers.google.com/web/tools/lighthouse/audits/speed-index))
  - [ ] Reduce render-blocking stylesheets ([learn more](https://developers.google.com/web/tools/lighthouse/audits/blocking-resources))
  - [ ] Unused CSS rules ([learn more](https://developers.google.com/speed/docs/insights/OptimizeCSSDelivery))
  - [ ] ~~Inefficient cache policy on static assets (Google Analytics)~~  won't fix
- **Progressive Web App 55**
  Will be a separate epic
- **Accessibility 83**
  - [x] Insufficient contrast ratio ([learn more](https://dequeuniversity.com/rules/axe/2.2/color-contrast?application=lighthouse))
  - [x] `<html>` does not have a `[lang]` attribute ([learn more](https://dequeuniversity.com/rules/axe/2.2/html-lang?application=lighthouse))
- **Best Practices 88**
  - [x] Use HTTP/2 for all resources ([learn more](https://developers.google.com/web/tools/lighthouse/audits/http2))
  - [ ] Manifest's `short_name`... ([learn more](https://developers.google.com/web/tools/lighthouse/audits/manifest-short_name-is-not-truncated))
- **SEO 89**
  - [ ] Legible font sizes ([learn more](https://developers.google.com/web/fundamentals/design-and-ux/responsive/#optimize_text_for_reading))

**Comment by CrocoDillon** [comment]
Might also include these from https://www.digitalocean.com/community/tutorials/how-to-set-up-nginx-with-http-2-support-on-ubuntu-16-04?#step-10-%E2%80%94-optimizing-nginx-for-best-performance

- [ ] Connection credentials caching
- [ ] HTTP Strict Transport Security
